csharp/beetlex/web.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <ServerGarbageCollection>true</ServerGarbageCollection>
+  </PropertyGroup>
```

Comment:
is this specific to `BeetleX` ? or is this `best-practices` in `C#` world ?

Reply:
BeetleX FastHttpApi framework is practices in C#

Reply:
Ah ok. You mean this is specific to `BeetleX`, ok.

I asked you that just since it could be accurate for all `C#` based **frameworks**

Reply:
You could use it for all c# frameworks but i believe setting it to false actually is more aggressive in memory reclaiming so depends on your needs